circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -72,7 +72,7 @@ enum GeneratorIndex {
     FUNCTION_LEAF,
     CONTRACT_DEPLOYMENT_DATA,
     CONSTRUCTOR,
-    CONSTRUCTOR_ARGS,
+    FUNCTION_ARGS,
```

Comment:
Again, wondering if Suyash can give insight into whether it is okay to use the same generator index for constructor args as normal function args.

Reply:
We now use two different generator indices.